core/src/test/scala/org/apache/spark/deploy/history/HistoryServerSuite.scala

```diff
@@ -349,6 +349,17 @@ class HistoryServerSuite extends SparkFunSuite with BeforeAndAfter with Matchers
 
   }
 
+  /**
+   * Verify that the security manager needed for the history server can be instantiated
+   * when `spark.authenticate` is `true`, rather than raise an `IllegalArgumentException`.
+   */
+  test("SecurityManagerStartsWithSecureShuffle") {
```

Comment:
nit: can you write a test description like the others in the class, instead of this camel-cased string?

Also spark.authenticate is not (just) for secure shuffle, so the string itself is not really accurate. Just mention spark.authenticate.